crates/admin/src/rest_api/components.rs

```diff
@@ -99,31 +97,19 @@ pub async fn modify_component<V>(
         ModifyComponentRequest,
     >,
 ) -> Result<Json<ComponentMetadata>, MetaApiError> {
-    state
+    let response = state
         .task_center
         .run_in_scope("modify-component", None, async {
```

Comment:
Minor note: An alternative to run_in_scope on every handler function was to add a middleware layer that does that for all calls.

Reply:
True. I'll create a follow-up for it.